projects/cds/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@spartacus/cds",
-  "version": "1.0.0-next.0",
+  "version": "2.0.0",
```

Comment:
I believe we should set `2.0.0-next.0` for all versions, will it break the logic?
(we should be able to run `ng update @spartacus/schematics@next` until 2.0 becomes the final)

Reply:
Done